go.mod

```diff
@@ -4,19 +4,21 @@ go 1.13
 
 require (
 	github.com/gobuffalo/flect v0.2.0
-	github.com/golang/protobuf v1.3.2 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/onsi/ginkgo v1.10.1
-	github.com/onsi/gomega v1.7.0
+	github.com/golang/protobuf v1.3.4 // indirect
```

Comment:
Could we not remove the //indirect imports?

Reply:
Out of scope of this PR.

Reply:
The above comment is to remove the new // indirect modules added here. 
Also, why was added new direct ones as well?

Reply:
If the other PR gets merged I will.

Reply:
Hi @Adirio you added here new entries. These ones are that one that I a suggestion you remove for we are able to move forward. I mean, can you just revert your changes in these files?

Reply:
After your chat I will approve your PR here and then, working on to clean it in the #1454

Reply:
As talked offline, we will handle this in #1454